telecarla_gui/test/gui/gui_parameter_test.cpp

```diff
@@ -143,7 +143,12 @@ TEST(GuiParameter, getVehicleStatusParameters_WhenValidTestFileWithoutStatus_Emp
 TEST(GuiParameter, getVehicleStatusParameters_WhenValidTestFile_CorrectParameters)
 {
     GuiParameter unit(vehicleStatusTestFilePath);
-    SDL_Rect expectedParameter{0, 720, 640, 200};
```

Comment:
Could you try if this also works `expectedParameter{.x = 0, .y = 0, .w = 0, .h = 0};`

Reply:
Works! 👍